acceptance/options/puppetserver_embedded_db.rb

```diff
@@ -0,0 +1,12 @@
+# We are eval'd in the scope of the acceptance framework's option-parsing
```

Comment:
I guess we should consider this now, hsqldb is the deprecated case, so testing on postgresql is the most important case :-).